tests/kafkatest/services/streams.py

```diff
@@ -215,8 +216,10 @@ def node(self):
 
     def pids(self, node):
         try:
-            return [pid for pid in node.account.ssh_capture("cat " + self.PID_FILE, callback=int)]
-        except:
+            pids = [pid for pid in node.account.ssh_capture("cat " + self.PID_FILE, callback=str)]
+            return [int(pid) for pid in pids]
+        except Exception, exception:
+            self.logger.debug(str(exception))
```

Comment:
We just log the exception on debug level. I do not know how much this will help, but it is better than nothing.